tests/containers/prepare_rootless_podman.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This is the preparation of rootless_podman
+#  * It switches to CGroup v2
+#  * It adds the uset to systemd-journal group
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    assert_script_run "usermod -a -G systemd-journal $testapi::username";
+
+    assert_script_run "sed -i '/^GRUB_CMDLINE_LINUX_DEFAULT/ s/\"\$/ systemd.unified_cgroup_hierarchy=1 \"/' /etc/default/grub";
```

Comment:
maybe you can use https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/bootloader_setup.pm#L1146